api/python/provisioner/inputs.py

```diff
@@ -1066,3 +1066,33 @@ def pillar_value(self):
                     'is_repo': False
                 }
             }
+
+
+@attr.s(auto_attribs=True)
+class SWUpgradeRemoveRepo(ParamDictItemInputBase):
+    _param_di = param_spec['swupgrade/repo']
+    release: str = ParamDictItemInputBase._attr_ib(
+        is_key=True,
+        descr="release version",
+        validator=attr.validators.matches_re(
+            "^[0-9]+\.[0-9]+\.[0-9]+\-[0-9]+$"),
+        converter=str
+    )
+
+    @property
+    def pillar_key(self):
+        # the local root pillar key for 'swupgrade' installation type
+        return self.release
+
+    @property
+    def pillar_value(self):
+        res = {
+            f"{repo}": None
```

Comment:
Done

Reply:
When I did this change it leads to exceptions. I think it is better to keep the first logic because I aligned it for `_macors.sls` and `upgrade.sls`. 
To illustrate the problem I post here the debug info:
```
 /usr/local/lib/python3.6/site-packages/provisioner/pillar.py(416)update()
-> if value is UNDEFINED:
(Pdb) p value
{'os': PRVSNR_UNDEFINED, 'cortx_iso': PRVSNR_UNDEFINED, '3rd_party': PRVSNR_UNDEFINED, 'python_deps': PRVSNR_UNDEFINED, '2.0.0-277': PRVSNR_UNDEFINED}
(Pdb) 
```
So we need to iterate over the dictionary to align `pillar.py` logic for each dictionary entry

Reply:
ok, let's keep that for now